doc/kb_sdk_make_ui.md

```diff
@@ -0,0 +1,169 @@
+# <A NAME="top"></A>![alt text](https://avatars2.githubusercontent.com/u/1263946?v=3&s=84 "KBase") [KBase SDK](../README.md)
+
+1. [Install SDK Dependencies](kb_sdk_dependencies.md)
+2. [Install SDK with Docker](kb_sdk_dockerized_install.md)
+3. [Create Module](kb_sdk_create_module.md)
+4. [Specify Module and Method(s)](kb_sdk_edit_module.md)
+5. [Implement Method(s)](kb_sdk_impl_methods.md)
+6. **Specify User Interface**
+7. [Locally Test Module and Method(s)](kb_sdk_local_test_module.md)
+8. [Register Module](kb_sdk_register_module.md)
+9. [Test in KBase](kb_sdk_test_in_kbase.md)
+10. [Complete Module Info](kb_sdk_complete_module_info.md)
+11. [Deploy](kb_sdk_deploy.md)
+
+### <A NAME="ui-widget"></A>6. Specify User Interface
+
+Control of Narrative interaction is accomplished in files in the ui/narrative/methods/<MyMethod> directory.
+
+#### A. Configure the input interface.
+
+The input options are specified in the "parameters" section of the spec.json file. In the following example, the user
+supplies two required parameters, an input name and an output name. By specifying a 'valid_ws_types' the user will
+be present a searchable dropdown of objects that match the specified type. By passing "is_output_name", the user is
+warned if a name will overwrite an existing object or if the name contains invalid characters.
```

Comment:
Also, the block with that output object will appear at the bottom of the app cell.